corehq/apps/hqadmin/views.py

```diff
@@ -440,6 +447,18 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class DomainAdminRestoreView(AdminRestoreView):
+    urlname = 'domain_admin_restore'
+
+    def dispatch(self, request, *args, **kwargs):
+        return TemplateView.dispatch(self, request, *args, **kwargs)
```

Comment:
The same issue would still exist. The problem is that this one restricts to domain admins but not necessarily superusers and the other restricts to superusers but not necessarily domain admins.